godot-core/src/meta/rpc_config.rs

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use crate::builtin::{Dictionary, StringName};
+use crate::classes::multiplayer_api::RpcMode;
+use crate::classes::multiplayer_peer::TransferMode;
+use crate::classes::Node;
+use crate::dict;
+use crate::meta::ToGodot;
+
+/// See [Godot documentation](https://docs.godotengine.org/en/stable/tutorials/networking/high_level_multiplayer.html#remote-procedure-calls)
+#[derive(Clone, Copy, Debug)]
+pub struct RpcConfig {
+    pub mode: RpcMode,
+    pub transfer_mode: TransferMode,
+    pub call_local: bool,
+    pub transfer_channel: u32,
+}
+
+impl Default for RpcConfig {
+    fn default() -> Self {
+        Self {
+            mode: RpcMode::AUTHORITY,
+            transfer_mode: TransferMode::UNRELIABLE,
+            call_local: false,
+            transfer_channel: 0,
+        }
+    }
+}
+
+impl RpcConfig {
+    /// Register `method` as a remote procedure call on `node`.
+    pub fn register(self, node: &mut Node, method: impl Into<StringName>) {
+        node.rpc_config(method.into(), &self.into_dictionary().to_variant());
+    }
+
+    /// Returns a [`Dictionary`] populated with the values required for a call to [`Node::rpc_config`].
+    pub fn into_dictionary(self) -> Dictionary {
+        dict! {
+            "mode": self.mode,
+            "transfer_mode": self.transfer_mode,
+            "call_local": self.call_local,
+            "transfer_channel": self.transfer_channel,
+        }
+    }
```

Comment:
According to [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-rpc-config), the key names here are incorrect.

Could you then also rename the struct fields to match the dict keys?